staging/src/k8s.io/apiserver/pkg/endpoints/filters/traces.go

```diff
@@ -30,11 +30,9 @@ func WithTracing(handler http.Handler, tp *trace.TracerProvider) http.Handler {
 	opts := []otelhttp.Option{
```

Comment:
WithTracing needs to be changed to `func WithTracing(handler http.Handler, tp trace.TracerProvider) http.Handler {` (without pointer to interface).

Reply:
updated